src/store/mutations.ts

```diff
@@ -73,6 +74,7 @@ const mutations: MutationTree<CalChartState> = {
     ) => StuntSheet;
     const currentSS = getSelectedStuntSheet(state);
     currentSS.dotTypes.push([new ContInPlace()]);
+    currentSS.dotAppearances.push(new DotAppearance());
```

Comment:
Does this mean that this just adds the same dot appearance for every dot type?

Reply:
Yep, for every new dot type the appearance is the same. Would a better idea be creating a different dot appearance depending on how many dot types are already in the stuntsheet?

Reply:
IMO, having just one appearance is great until we add UI for giving the user the ability to control individual dot appearances.